build.make

```diff
@@ -143,18 +143,19 @@ $(CMDS:%=push-multiarch-%): push-multiarch-%: check-pull-base-ref build-%
 	fi; \
 	pushMultiArch () { \
 		tag=$$1; \
-		echo "$$build_platforms" | tr ';' '\n' | while read -r os arch suffix base_image; do \
+		echo "$$build_platforms" | tr ';' '\n' | while read -r os arch suffix base_image addon_image; do \
 			docker buildx build --push \
-				--tag $(IMAGE_NAME):$$arch-$$os-$$tag \
+				--tag $(IMAGE_NAME):$$arch-$$base_image-$$os-$$tag \
```

Comment:
`base_image` can be empty, leading to a tag with double `--`. Not sure whether it matters, but it would be nicer to avoid this.

Reply:
correct, I'll mark this as a draft PR because I think it's not ready yet, thanks for checking

Reply:
fixed